nz.scuttlebutt.Patchwork.json

```diff
@@ -0,0 +1,177 @@
+{
+    "app-id": "nz.scuttlebutt.Patchwork",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "branch": "stable",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "command": "run.sh",
+    "separate-locales": false,
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--socket=x11",
+        "--filesystem=~/.ssb:create",
+        /* For uploading avatar picture; Electron doesn't use portals: */
+        "--filesystem=home:ro"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "NPM_CONFIG_LOGLEVEL": "info"
+        }
+    },
+    "modules": [
+        {
+            "name": "nodejs",
+            "cleanup": [
+                "/include",
+                "/share",
+                "/app/lib/node_modules/npm/changelogs",
+                "/app/lib/node_modules/npm/doc",
+                "/app/lib/node_modules/npm/html",
+                "/app/lib/node_modules/npm/man",
+                "/app/lib/node_modules/npm/scripts"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://nodejs.org/dist/v8.11.2/node-v8.11.2.tar.xz",
+                    "sha256": "539946c0381809576bed07424a35fc1740d52f4bd56305d6278d9e76c88f4979"
+                }
+            ]
+        },
+        {
+            "name": "xkbfile",
+            "cleanup": [
+                "/include",
+                "/lib/*.la",
+                "/lib/pkgconfig"
+            ],
+            "config-opts": ["--disable-static"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.x.org/releases/individual/lib/libxkbfile-1.0.9.tar.bz2",
+                    "sha256": "51817e0530961975d9513b773960b4edd275f7d5c72293d5a151ed4f42aeb16a"
+                }
+            ]
+        },
+        {
+            "name": "patchwork",
+            "build-options" : {
+                "env": {
+                    /* Need this for electron-download to find the cached electron binary */
+                    "electron_config_cache": "/run/build/patchwork/npm-cache"
+                }
+            },
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/ssbc/patchwork.git",
+                    "tag": "v3.9.0",
+                    "commit": "05199c5427b6df72e01ba59a2d57db4d6d610c2b",
+                    "dest": "main"
+                },
+                {
+                    "type": "file",
+                    "only-arches": ["x86_64"],
```

Comment:
If it works only on x86_64, you also need to add `flathub.json` file with `{"only-arches": ["x86_64"]}`.

Reply:
It _should_ work on other architectures, but these prebuilds were only available for x86_64. I see the build failed without the prebuilds, so I guess I'll try harder to get it to work...